src/auth/AxiosJwtAuthService.js

```diff
@@ -197,8 +197,8 @@ class AxiosJwtAuthService {
    * @returns {Promise<UserData>|Promise<null>} Resolves to the user's access token if they are
    * logged in.
    */
-  async fetchAuthenticatedUser() {
-    const decodedAccessToken = await this.jwtTokenService.getJwtToken();
+  async fetchAuthenticatedUser(options = {}) {
+    const decodedAccessToken = await this.jwtTokenService.getJwtToken(options.forceRefresh || false);
 
     if (decodedAccessToken !== null) {
```

Comment:
@davidjoy Do we need to unset the user here if `forceRefresh` is `true` and `decodedAccessToken` is `null`? Otherwise, it causes an extra redirect since the returned value will be a valid token.

Reply:
Ahh, good point, yes, I think you'll want to do that.  Otherwise the app will continue to have a value returned by `getAuthenticatedUser`.  If the decoded token is `null`, I think we want to `this.setAuthenticatedUser(null)`

Reply:
Done.